geoportal/c2cgeoportal_geoportal/scaffolds/create/ci/trigger

```diff
@@ -9,6 +9,12 @@ import sys
 def main():
     parser = argparse.ArgumentParser(description="Publish Docker images")
     parser.add_argument("--image", dest="images", action="append", help="The image to be exported")
+    parser.add_argument(
+        "--image-stream-name",
+        dest="imagestreams",
+        action="append",
+        help="The image stream name template, can have the values: {version}, {image}, {env}, {version_only}.",
```

Comment:
The image stream names can use template values: {version}, {image}, {env}, {version_only}. Default to {version}-c2cgeoportal-{image}:{version}.